aries-site/src/pages/components/skeleton.mdx

```diff
@@ -16,15 +16,11 @@ import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
 </Example>
 
 ### When to use
-Skeletons are used on all pages to provide a visual layout of the content being loaded.  They help users perceive a loading period as being shorter in duration.
+- The interface is content-rich and has asynchronous loading, in order to provide the user with a sense of the page structure and to minimize layout shifts when content is loaded in.
 
 ### When not to use
```

Comment:
Since this is directly related to "When to use," should this be a child/subsection under "when to use" as opposed to a peer?

```suggestion
#### When not to use
```